docs/docs/manual/cellediting.md

```diff
@@ -103,52 +103,95 @@ You can also export the currently identified clusters as a JSON file, or close t
 
 ### Clustering methods {#clustering-methods}
 
-You don’t need to understand the details behind each clustering method to apply them successfully to your data. The order in which these methods are presented in the interface and on this page is the order we recommend - starting with the most strict rules and moving to the most lax, which require more human supervision to apply correctly. 
+You don’t need to understand the details behind each clustering method to apply them successfully to your data.
 
-The clustering pop-up window offers you a variety of clustering methods:
+The order in which these methods are presented in the interface and on this page is the order we recommend - starting with the most strict rules and moving to the most lax, which require more human supervision to apply correctly.
 
-*   key collision
-    *   fingerprint
-    *   ngram-fingerprint
-    *   metaphone3
-    *   cologne-phonetic
-    *   Daitch-Mokotoff
-    *   Beider-Morse
-*   nearest neighbor
-    *   levenshtein
-    *   ppm
+The clustering pop-up window offers you two categories of clustering methods: 6 different key collision clustering methods and 2 nearest neighbour clustering methods. 
+
+- [Key collision](#key-collision)
```

Comment:
```suggestion
- [Key Collision](#key-collision)
```

Could you capitalise every summary items? Like the title there are below. There is more than 10 that could be capitalized from what I saw.